### ISSUE TITLE
Does this library implement multipart?

### ISSUE THREAD

**Issue by lewibs** [ryanto/next-s3-upload#178]
Im trying to upload multi GB files and found this library. I cant seem to find any reference to multipart so I'm asking here.

**Comment by AminDhouib** [comment]
No, it issues a single presigned PUT per file, so there's no multipart and no resume. At multi-GB that means one dropped connection restarts the whole transfer, and the repo hasn't had a commit since late 2024, so it isn't going to grow the feature. If you still need it, upup (https://useupup.com) does chunked resumable uploads to S3 and other S3-compatible providers, and `@upupjs/next` gives you the App Router and Pages Router route handlers. I work on upup.
